tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -2665,13 +2666,14 @@ def test_snuba_unsupported_filters(self):
                 )
                 assert mock_query.call_count == 0
 
+    @pytest.mark.skip(reason="Need to fix")
     @patch(
         "sentry.search.snuba.executors.GroupAttributesPostgresSnubaQueryExecutor.query",
         side_effect=GroupAttributesPostgresSnubaQueryExecutor.query,
         autospec=True,
     )
     @override_options({"issues.group_attributes.send_kafka": True})
-    def test_snuba_query_title(self, mock_query):
+    def xtest_snuba_query_title(self, mock_query):
```

Comment:
oh this maybe will get flagged in a different way by pytest or unittest -- can leave the method name the same

Reply:
@asottile-sentry whoops, I didn't mean to put that there